edx_solutions_api_integration/courses/tests.py

```diff
@@ -1367,11 +1363,11 @@ def test_courses_users_list_get_attributes(self, store):
             grade_summary=json.dumps(grade_summary)
         )
 
-        StudentProgress.objects.update_or_create(
+        Aggregator.objects.update_or_create(
             user=user,
             course_id=course.id,
             defaults={
-                'completions': user_completions,
+                'earned': user_completions,
```

Comment:
This needs a course_key (instead of course_id) and block_key (the UsageKey of the root course block).